src/ast/mod.rs

```diff
@@ -7289,6 +7310,16 @@ pub enum SqlOption {
         range_direction: Option<PartitionRangeDirection>,
         for_values: Vec<Expr>,
     },
+
+    Comment(CommentDef),
+
+    TableSpace(TablespaceOption),
+
+    // Advanced parameter formations
+    // UNION  = (tbl_name[,tbl_name]...)
+    // ENGINE = ReplicatedMergeTree('/table_name','{replica}', ver)
+    // ENGINE = SummingMergeTree([columns])
```

Comment:
```suggestion
    /// An option representing a key value pair, where the value is a parenthesized list and with an optional name.
    ///
    /// ```sql
    /// UNION  = (table1, table2)
    /// ENGINE = ReplicatedMergeTree('/table_name','{replica}', ver)
    /// ENGINE = SummingMergeTree(column1, column2)
    /// ```
```